met-etl/migrations/versions/2c7d5ce6e117_analytics_db.py

```diff
@@ -0,0 +1,210 @@
+"""analytics_db
+
+Revision ID: 2c7d5ce6e117
+Revises: 
+Create Date: 2022-07-22 16:17:22.886477
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2c7d5ce6e117'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('engagement',
+    sa.Column('engagement_id', sa.Integer(), nullable=False),
+    sa.Column('engagement_name', sa.String(length=50), nullable=True),
+    sa.Column('start_date', sa.DateTime(), nullable=True),
```

Comment:
Can we remove engagement prefix from the column names?
like since its in engagement table , its kind of assumed that id is engagement id...
a select statement will also be shorter , select id from engagement instead of select engagementd from enegagement

Not 100% fixated on this..we can discuss if you have different opinons...

Reply:
this will apply across all tables

Reply:
Addressed